src/utils/loggable.py

```diff
@@ -0,0 +1,155 @@
+import logging
```

Comment:
Is this an existing module? It seems kind of familiar. Can we add it to the flatpak-builder manifest and import it instead?

Reply:
It didn't exist, I wrote this by myself :-)